moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -1,6 +1,14 @@
+export const filterNullValues = (value) => {
+  if (!value) {
+    return "-";
```

Comment:
@tillyw would you remove the `-` and just return an empty string `""` ? i think that will be preferred for folks working with this data in a spreadsheet editor.